build.gradle

```diff
@@ -48,10 +48,12 @@ dependencies {
 
     compileOnly 'com.destroystokyo.paper:paper-api:1.16.4-R0.1-SNAPSHOT'
     compileOnly 'net.md-5:bungeecord-api:1.16-R0.4-SNAPSHOT'
+
     implementation 'org.jetbrains:annotations:20.1.0'
     implementation 'com.google.code.gson:gson:2.8.6'
     implementation 'io.github.classgraph:classgraph:4.8.92'
     implementation 'com.github.seancfoley:ipaddress:5.3.3'
+    implementation 'com.squareup.okhttp3:okhttp:4.9.0'
```

Comment:
Surely there are other ways to make HTTP requests aside from shading an entire new dependency?

Reply:
Vlad or koda (have to check which) said to use this

Reply:
Guava provides a reasonable HTTP thing - the [GET method is only 40 lines long](https://gist.github.com/tfnico/5926374).

Reply:
I'll have to make sure it can provide everything that I want/use, but I'll take it under strong consideration.